examples/create-job-by-nodename.js

```diff
@@ -0,0 +1,36 @@
+
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+
+function getJobNodeNames(nameSpace, kubernetes) {
+  return kubernetes.jobs.list(nameSpace).map(function(job){
+    return job.spec.template.spec.node_name
+  })
+}
+
+export default function () {
+  const kubernetes = new Kubernetes({
+    // config_path: "/path/to/kube/config", ~/.kube/config by default
+  })
+  const namespace = "default"
+  const jobName = "new-nodename-job"
+  const nodeName = "my-node-name"
```

Comment:
Instead of hard coding the node name here we could use the node list function
```
const nodes = kubernetes.nodes.list()
const nodeName = nodes[0].name 
```

Reply:
fixed by https://github.com/grafana/xk6-kubernetes/pull/29/commits/e851a4bce654f018a027401dd5e6314e41832b62